### DISCUSSION TITLE
Arduino IDE 2rc - upload does nothing

### DISCUSSION THREAD

**Discussion by apws** [SpenceKonde/ATTinyCore#634] [General]
I am trying also this new IDE 2 and while old one works perfectly with installed ATTinyCore and megaTinyCore, the new one does nothing after clicking Upload. I had few minutes ago some beta.12 version where it also not worked and here I reselected board by clicking on toolbar "Select other board and port" and then both compiling AND uploading started to work. But after reinstalling rc2 over beta12, uploading again doesnt work. This seems to be some hard issue of IDE2 though, I will report it there... but what is surprising too is that also "Burn bootloader" is not working with error "Error while burning the bootloader: Property 'bootloader.tool.' is undefined" 
(sorry I am quite new in arduino ides and dont know if state of new IDE is well known issue for you...)

**Comment by SpenceKonde** [comment]
Have you installed manually or via board manager? Last I heard the all the Arduino IDE 2 versions were incompatible with manually installed cores. 

In any event, this is not a bug in the core, it is a bug in the Arduino IDE 2RC that should be reported to the Arduino IDE team. The default assumption with arduino IDEversions should always be that a new version is totally hosed and should not be used until proven otherwise

**Reply by apws** [reply]
via board manager, ya, I reported this to them ... okay

**Comment by SpenceKonde** [comment]
What output do you see when you check verbose upload and attempt to upload. 

That'll at least tell us what it's doing wrong/
